service/docs/source/geopmsession.1.rst

```diff
@@ -2,6 +2,44 @@
 geopmsession(1) -- Command line interface for the GEOPM service batch read features
 ===================================================================================
 
+Synopsis
+--------
+
+Read a signal
+~~~~~~~~~~~~~
+
+.. code-block:: none
+
+    echo "SIGNAL_NAME DOMAIN DOMAIN_IDX" | geopmsession
+
+Read a signal at a specific period
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: none
+
+    geopmsession -p | --period
```

Comment:
This renders as `geopmsession -p | --period` which is confusing since the "read a signal" uses `|` to mean a pipe and here it is supposed to mean "or".